fh-messaging/lib/mongo_functions/mbaas_reduce_apprequestsdest.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable*/
+//ts and emit are used for testing purposes
+var ts;
+var emit;
+function mbaas_reduce_apprequestsdest(k, mbaasMetrics) {
+  // Take the domain and ts in the first vals entry as the reduced elements domain and ts
+  var result = {};
+
+
+  function forEach(iterable, cb) {
+    for (var i in iterable) {
+      if (iterable.hasOwnProperty(i)) {
+        cb(iterable[i], i);
+      }
+    }
+  }
+
+  forEach(mbaasMetrics, function(appMetrics) {
```

Comment:
This gets executed in the mongo environment so i didn't get any errors only a failed execution but anytime I tried reference `forEach` or `reduce` the mapreduce failed